### ISSUE TITLE
Add requirements.txt

### ISSUE THREAD

**Issue by michaelpginn** [macramole/CHAFile#1]
It would be nice if the repo included a requirements.txt file that lists the packages needed, since some of them such as `lexical-diversity` are non-standard.

**Comment by macramole** [comment]
That's a great idea, thanks.

I would actually like to publish it on pip but I haven't found the time to
do so yet.


--
leandrogarber.info

**Comment by jonesetc** [comment]
@macramole Would you be open to a PR that gets it into a package with poetry? My partner is a is working on her dissertation using some of the CHILDES transcripts. I helped her get this code working by just vendoring it, but would be happy to help you get it packaged up so it's easier for her and others to incorporate in the future.

**Comment by macramole** [comment]
Yes, that would be great for sure. I don't use poetry myself, I usually use
pip or conda, but a package would be great. I guess I can still add pip
support in the future as well.


--
leandrogarber.info
